NoSQL/README.md

```diff
@@ -54,29 +54,79 @@ You can also use the Oracle NoSQL Command Line Interface (CLI). Start the
-  Admin [admin1]     Status: RUNNING,MASTER
-  Rep Node [rg1-rn1] Status: RUNNING,MASTER sequenceNumber:49 haPort:5006
+   Pinging components of store kvstore based upon topology sequence #14
+   10 partitions and 1 storage nodes
+   Time: 2021-12-20 12:56:33 UTC   Version: 20.3.19
+   Shard Status: healthy:1 writable-degraded:0 read-only:0 offline:0 total:1
+   Admin Status: healthy
+   Zone [name=KVLite id=zn1 type=PRIMARY allowArbiters=false masterAffinity=false]   RN Status: online:1 read-only:0 offline:0
+   Storage Node [sn1] on dcbd8ff4f07c:5000    Zone: [name=KVLite id=zn1 type=PRIMARY allowArbiters=false masterAffinity=false]    Status: RUNNING   Ver: 20.3.19 2021-09-29 04:04:01 UTC  Build id: b8acf274b357 Edition: Community
+        Admin [admin1]          Status: RUNNING,MASTER
+        Rep Node [rg1-rn1]      Status: RUNNING,MASTER sequenceNumber:50 haPort:5003 available storage size:1023 MB
+
 
   kv-> put kv -key /SomeKey -value SomeValue
   Operation successful, record inserted.
   kv-> get kv -key /SomeKey
   SomeValue
-  kv->
+  kv-> exit
+```
+
+You can also use the Oracle SQL Shell Command Line Interface (CLI). Start the
+following container (keep container `kvlite` running):
+
+```shell
+$ docker run --rm -ti --link kvlite:store oracle/nosql:20.3 \
+  java -jar lib/sql.jar -helper-hosts store:5000 -store kvstore
+
+  sql-> show tables
+  tables
+    SYS$IndexStatsLease
+    SYS$MRTableAgentStat
+    SYS$MRTableInitCheckpoint
+    SYS$PartitionStatsLease
+    SYS$SGAttributesTable
+    SYS$StreamRequest
+    SYS$StreamResponse
+    SYS$TableStatsIndex
+    SYS$TableStatsPartition
+  sql-> exit
+
 ```
 
+## Oracle NoSQL Database Proxy
+
+The Oracle NoSQL Database Proxy is a middle-tier component that lets the Oracle NoSQL Database drivers communicate with the Oracle NoSQL Database cluster. 
+The Oracle NoSQL Database drivers are available in various programming languages that are used in the client application.
+
+The Oracle NoSQL Database Proxy is a server that accepts requests from Oracle NoSQL Database drivers and processes them using the Oracle NoSQL Database. 
+The Oracle NoSQL Database drivers can be used to access either the Oracle NoSQL Database Cloud Service or an on-premises installation via the Oracle NoSQL Database Proxy. 
+Since the drivers and APIs are identical, applications can be moved between these two options. 
+
+You can deploy a docker Oracle NoSQL Database store first for a prototype project, and move forward to Oracle NoSQL Database cluster for a production project.
+
+Here is a snippet showing the connection from a Node.js program, the HTTP proxy is automatically started in this docker image
+
+````
+    /*
+     * EDIT: if the endpoint does not reflect how the Proxy
+     * Server has been started, modify it accordingly.
+     */
+    return new NoSQLClient({
+        serviceType: ServiceType.KVSTORE,
+        endpoint: 'docker-host-container-nosql:8081'
+    });
+````
```

Comment:
If you're using a code fence, you don't need to indent the content as well, so please remove the spaces from the lines inside the fence.